api_test.go

```diff
@@ -1,11 +1,24 @@
-package gogpt
+package gogpt_test
 
 import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
 	"testing"
+	"time"
+
+	. "github.com/sashabaranov/go-gpt3"
+)
+
+const (
+	testAPIToken = "this-is-my-secure-token-do-not-steal!!"
 )
 
 func TestAPI(t *testing.T) {
```

Comment:
Currently, running `make test` fails due to this test. Maybe we should mark it as a [long test](https://stackoverflow.com/questions/55180613/how-do-i-write-test-in-go-that-make-use-of-the-short-flag-and-can-it-be-combine), and only run short tests in the `make test`?

Reply:
@sashabaranov What issues are you running into with this test? It's been running fine on mine

Reply:
@RobotSail this test requires `.openai-token` to be tested against real API. I suggest we disable this test to run by default and only run your new awesome one with a mock server!

Reply:
@sashabaranov I set this test to be skipped in short mode. In the long-term, I think we should migrate the other tests in short mode to also have mocked methods (I don't have experience with the other APIs, so I didn't include them in this PR).
My only concern with replacing all of the current tests with our mocked ones is the accuracy of the client.

One workaround to this could be having an OpenAI token set in the repo, and then running tests against the API after the runs are approved in order to keep costs down.

Reply:
Awesome work!

>  One workaround to this could be having an OpenAI token set in the repo, and then running tests against the API after the runs are approved in order to keep costs down.

I've contacted @openai about the test token for the CI, but they didn't reply back.

Reply:
@sashabaranov We've also asked them about having test endpoints to be able to test without burning tokens, they said that it was something that they were considering, but I've yet to see it.